shared/folders/render.desktop.js

```diff
@@ -45,6 +50,10 @@ class Render extends Component<void, Props, State> {
   }
 
   render () {
+    if (this.props.allowComingSoon && !flags.tabFoldersEnabled) {
```

Comment:
To add another point:

Dumb components have been mostly pure, and these feature flags are not pure, there is some external system state that causes these to render differently. We should strive to keep these pure.